servicecontrol/servicecontrol-instances/hardware.md

```diff
@@ -4,28 +4,28 @@ summary: Hardware recommendations for running ServiceControl instances
+This article provides recommendations and performance benchmarks to help select resources for a ServiceControl production environment.
 
 ## General recommendations
 
 * A dedicated production server for installing ServiceControl instances (Primary, Audit, and Monitoring).
 * A minimum of 16 GB of RAM (excluding RAM for OS and other services).
 * 2 GHz quad core CPU or better.
-* A dedicated disk for ServiceControl databases that is separate from the operating system.
+* A dedicated disk for ServiceControl databases (not the disk where the operating system is installed).
 
 ### Scaling ServiceControl
 
-When possible, scaling *up* a single machine to handle system load is recommended. When scaling up is not an option, ServiceControl can be scaled *out* by partitioning audit processing between multiple instances. See [Multiple ServiceControl Instances](remotes.md) for more details.
+When possible, scaling *up* a single machine to handle system load is recommended. When scaling up is not an option, ServiceControl may be scaled *out* by partitioning audit processing between multiple instances. See [Multiple ServiceControl Instances](remotes.md) for more details.
 
 ### Ongoing server performance monitoring
 
-The requirements for a server hosting ServiceControl can change over time as the system evolves. It's important to continuously monitor the CPU, RAM, disk I/O, and network I/O for the server running ServiceControl to ensure adequate resources are available for overall system health.
+The requirements for a server hosting ServiceControl may change over time as the system evolves. It's important to continuously monitor the CPU, RAM, disk I/O, and network I/O for the server running ServiceControl to ensure adequate resources are available for overall system health.
 
-Real disk, CPU, RAM, and network performance can be monitored with the Windows Resource Monitor and/or Windows Performance counters.
+Disk, CPU, RAM, and network performance may be monitored using the Windows Resource Monitor and/or Windows Performance counters.
 
 ### Storage recommendations
 
-* Store ServiceControl data on a dedicated disk. This makes low-level resource monitoring easier and ensures different applications are not competing for storage IOPS.
+* Store ServiceControl data on a dedicated disk. This makes low-level resource monitoring easier and ensures applications are not competing for storage IOPS.
 * Store multiple ServiceControl databases on separate physical disks to prevent multiple instances competing for the same disk resources.
 * Disable disk write caching (read caching can remain enabled) to prevent data corruption if the (virtual) server or disk controller fails. This is a general best practice for databases.
 * [Database paths](/servicecontrol/creating-config-file.md#host-settings-servicecontroldbpath) should be located on disks suitable for low latency write operations (e.g. fiber, solid state drives, raid 10), with a recommended IOPS of at least 7500.
```

Comment:
Futher up we refer to "ServiceControl data" and "ServiceControl databases", yet here (and on line 57), we switch to "Database paths". Should we be constistent?

Reply:
I actually didn't know enough to know they were referring to the same thing. If they are, then yes, definitely, we should be consistent!

Reply:
As I originally wrote this, with data I meant everything in the DB Path which is not only databases. Maybe we can use CONFIGURED DATABASE PATH, but we are not consistent in SCMU either. For me servicecontrol STORAGE path is a better generic name that would include databases, indexes, logs and failed messages.

Reply:
Let's leave it for now. As discussed, these docs are up for review soon.